src/ixdat/calculators/ms_calculators.py

```diff
@@ -0,0 +1,829 @@
+        if T is None:
+            T = self.T
+        if p is None:
+            p = self.p
+        pi = np.pi
+
+        # TODO: make it so that DYNAMIC_VISCOSITIES[gas] can just be a float if someone
+        #   enters it without having access to the temperature-dependent values.
+        if T < DYNAMIC_VISCOSITIES[gas][0, 0] or T > DYNAMIC_VISCOSITIES[gas][-1, 0]:
+            warnings.warn(
+                "Insufficient data in constants.py to appropriately estimate "
+                f"the dynamic viscosity for {gas} at temperature: {T}K",
+                stacklevel=2,
+            )
+        _eta_v = DYNAMIC_VISCOSITIES[gas][:, 1]  # list of known eta(T) for 'gas'
+        _eta_T = DYNAMIC_VISCOSITIES[gas][:, 0]  # list of paired Ts for eta(T)
+
+        eta = np.interp(T, _eta_T, _eta_v)  # dynamic viscosity of gas at T in [Pa*s]
+
+        s = MOLECULAR_DIAMETERS[gas]  # molecule diameter in [m]
+        m = MOLAR_MASSES[gas] * 1e-3 / AVOGADRO_CONSTANT  # molecule mass in [kg]
+
+        d = ((w_cap * h_cap) / pi) ** 0.5 * 2
+        # d = 4.4e-6  #used in Henriksen2009
+        a = d / 2
+        p_1 = p
+        lambda_ = d  # defining the transitional pressure
+        # ...from setting mean free path equal to capillary d
+        p_t = BOLTZMANN_CONSTANT * T / (2**0.5 * pi * s**2 * lambda_)
+        p_2 = 0
+        p_m = (p_1 + p_t) / 2  # average pressure in the transitional flow region
+        v_m = (8 * BOLTZMANN_CONSTANT * T / (pi * m)) ** 0.5
+        # a reciprocal velocity used for short-hand:
+        nu = (m / (BOLTZMANN_CONSTANT * T)) ** 0.5
+
+        # ... and now, we're ready for the capillary equation.
+        #   (need to turn of black and flake8 for tolerable format)
+        # fmt: off
+        #   Equation 4.10 of Daniel Trimarco's PhD Thesis:
+        N_dot = (                                                               # noqa
+            1 / (BOLTZMANN_CONSTANT * T) * 1 / l_cap * (                        # noqa
+                (p_t - p_2) * a**3 * 2 * pi / 3 * v_m + (p_1 - p_t) * (         # noqa
+                    a**4 * pi / (8 * eta) * p_m  + a**3 * 2 * pi / 3 * v_m * (  # noqa
+                        (1 + 2 * a * nu * p_m / eta) / (                        # noqa
+                        1 + 2.48 * a * nu * p_m / eta                           # noqa
+                        )                                                       # noqa
+                    )                                                           # noqa
+                )                                                               # noqa
+            )                                                                   # noqa
+        )                                                                       # noqa
+        # fmt: on
+        n_dot = N_dot / AVOGADRO_CONSTANT
+        return n_dot
+
+    # --- METHODS WHICH HAVE BEEN MOVED TO `Calculator` CLASSES ---- #
```

Comment:
copy & paste mistake here? seems like all the below should be in the techniques module, right? And deleted here.
Might be there, as well, I did not double check now.

Reply:
This is from 0.2.5... 
But yeah I'm thinking we delete everything with `hard_deprecation_release=0.3.0` before making the release!